src/i18n/i18n.jl

```diff
@@ -82,7 +85,8 @@ const language_codes_registered = Ref{Dict{AbstractLanguage,Vector{String}}}(
           PTTGerman.GermanGermanyColloquial() => ["de_colloq", "de_de_colloq"],
           PTTRussian.RussianRU() => ["ru","ru_ru"],
           PTTSpanish.SpanishES() => ["es", "es_es"],
-          PTTSpanish.SpanishColloquial() => ["es_colloq", "es_es_colloq"]
+          PTTSpanish.SpanishColloquial() => ["es_colloq", "es_es_colloq"],
+          PTTChinese.ChineseZH() => ["zh", "zh-cn"]
```

Comment:
This is a minor detail, but all other language codes use underscores instead of dashes.
```suggestion
          PTTChinese.ChineseZH() => ["zh", "zh_cn"]
```